### DISCUSSION TITLE
Feature request: a feature requests space

### DISCUSSION THREAD

**Discussion by jsamr** [community/community#4502] [Other Feature Feedback, Questions, & Ideas]
Often, the issue tracker is also used as a backlog for feature requests. However, numerous tools will use the number of open issues as a quality metric for the repository, leading to disadvantaging repositories with lots of open issues. Examples:

- [Npm.io uses a ratio of open / close issues to compute a maintenance score](https://npms.io/about);
- [React Native Directory will add a penalties for projects with more than 75 open issues](https://reactnative.directory/directory-score).

This can lead to an incentive to close issues and lose track of feature requests.

## Proposal

- A distinct **feature requests space** accessible via a tab;
- Feature requests, contrary to discussions, can be referenced in commits;
- Feature requests can be used in the project board.
- Feature requests can be templated or "formalized" (alpha feature) to guide the user into submission.

**Comment by huehnerlady** [comment]
I thought this is what you can use Discussions for? It even has a voting feature? 🤔 
If then something is actually don, you could convert it to an issue?

**Reply by jsamr** [reply]
I didn't consider discussions because they lack the below qualities: 

- Reference discussions in commits;
- Constraint discussions with templates / forms;
- Track discussions in project board (not 100% sure about that one though).

Moreover, if you can convert a discussion into an issue, well, you get back to the first problem: number of open issues is often a quality metric.

**Comment by gnprice** [comment]
I don't disagree with this feature request, but I wanted to note that that metric some tools use is a terrible idea for multiple reasons.

The biggest one is that it creates an incentive to routinely close issues even when they're bug reports, and when there's no reason to think they've been fixed. Many projects comply with this incentive by having a bot declare issues "stale" and close them just because they haven't gotten any replies for a few weeks. The result is often that lots of bugs turn into closed issues without ever getting fixed, which just makes the issue tracker worse.

So even in a future where this feature request is implemented, that metric will still be a terrible idea, and I hope that people who are creating those harmful incentives by scoring repos that way will reconsider.
